pkg/kudoctl/cmd/upgrade.go

```diff
@@ -8,9 +8,8 @@ import (
 	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
 	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
 	"github.com/kudobuilder/kudo/pkg/kudoctl/util/repo"
-	util "github.com/kudobuilder/kudo/pkg/util/kudo"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/resources"
```

Comment:
could we move it one level up so it's `pkg/kudoctl/util/resources` ?

Reply:
Let's consolidate the footprint in this PR so it's focused on the new package, and then we can immediately open a refactoring that moves util out and gets rid of it entirely, wdyt @zen-dog?